src/main/java/ui/page/checkoutStages/MainPage.java

```diff
@@ -0,0 +1,16 @@
+package ui.page.checkoutStages;
+
+import com.codeborne.selenide.SelenideElement;
+import org.openqa.selenium.By;
+
+import static com.codeborne.selenide.Selenide.$;
+
+public class MainPage {
+    private SelenideElement productPhoto = $(By.cssSelector("#add_to_cart"));
+
```

Comment:
"#add_to_cart" it's id, not selector.
 In addition,   "By.cssSelector" it's usage of selenium, need to rewrite with usage selenide